src/core/emergency-action.c

```diff
@@ -20,11 +20,12 @@
 ***/
 
 #include <sys/reboot.h>
-#include <linux/reboot.h>
 
 #include "bus-error.h"
 #include "bus-util.h"
 #include "emergency-action.h"
+#include "raw-reboot.h"
+#include "reboot-util.h"
```

Comment:
This is not necessary.

Reply:
emergency-action.c calls update_reboot_parameter_and_warn() which is defined in reboot-util.h.

emergency-action.c also calls raw_reboot() which is defined in raw-reboot.h.

Hence it really needs both includes.

Reply:
Indeed. I thought that the functions provided by reboot-util.h are new. But this is not correct. Sorry for my misunderstanding.